receipt/SaleReceipt.tpl

```diff
@@ -931,19 +987,19 @@ table.payments td.label {
 			{% endif %}
 		{% endif %}
 
-		<td data-automation="lineItemQuantity" class="quantity">
-			{% if options.per_line_subtotal and options.discounted_line_items and Line.calcLineDiscount != 0 and not parameters.gift_receipt %}
-				<span class="strike">{{Line.unitQuantity}} x {{Line.unitPrice|money}}</span>
-			{% endif %}
-			{{Line.unitQuantity}}
-			{% if options.per_line_subtotal and not parameters.gift_receipt %} x
-				{% if options.discounted_line_items %}
-					{{ divide(Line.displayableSubtotal, Line.unitQuantity)|money }}
-				{% else %}
-					{{Line.displayableUnitPrice|money}}
+			<td data-automation="lineItemQuantity" class="quantity">
```

Comment:
thought: Indentation seems wrong, maybe revert these changes

Reply:
Indenting should now be fixed.